sbi/inference/snl/snl.py

```diff
@@ -53,6 +54,10 @@ def __init__(
         perform inference on the posterior p(theta | x0).
         :param neural_likelihood: Conditional density estimator q(x | theta) in the form of an
         nets.Module. Must have 'log_prob' and 'sample' methods.
+        simulation_batch_size (int): how many parameter sets the simulator takes and converts to data x at
```

Comment:
Docstrings for SNL, SNPE, SRE are still old style and should be updated

Reply:
I would phrase it diffenrently, e.g., "the number of parameter sets the simulator takes ... "

Reply:
Also remember to remove types as agreed in recent issue #75.